cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -52,6 +56,12 @@ public final class DataprocUtils {
   public static final String CDAP_GCS_ROOT = "cdap-job";
   public static final String CDAP_CACHED_ARTIFACTS = "cached-artifacts";
   public static final String WORKER_CPU_PREFIX = "Up to";
+  public static final Path CACHE_DIR_PATH = Paths.get(System.getProperty("java.io.tmpdir"),
```

Comment:
Does this still needs to be java.io.tmpdir?

Reply:
Discussed offline.